### PR TITLE
Code comments translated

### PR DESCRIPTION
**Pull request by aingerusanchez**
Traducidos los comentarios del código de los ejemplos.

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] My article does not contain shortened URLs or affiliate links.

### CODE REVIEW

File: guide/spanish/java/variables/index.md [hunk 1]
```diff
@@ -17,32 +17,30 @@ Es posible que haya notado que el **tipo de envoltorio** consiste en tipos escri
...
 ```java
-// Initialise the variable of Primitive Data Type 'int' to store the value 10 
+ // Inicializa la variable primitiva del tipo 'int' para almacenar el valor 10
  int i = 10; 
  double amount = 10.0; 
  boolean isOpen = false; 
  char c = 'a'; // Note the single quotes 
  
- //Variables can also be declared in one statement, and assigned values later. 
+ // Las variables también pueden ser declaradas en una sentencia, y asignarles el valor en otra
```
Comment (thecodingaviator) [inline guide/spanish/java/variables/index.md]: This sentence looks a bit weird to me. Can one of @freeCodeCamp/spanish-language-editors confirm this is okay or suggest a change?

Comment (tomaspozo) [reply guide/spanish/java/variables/index.md]: I think it is understandable. This is how I would translate it in order to respect more the original:

`//Las variables tambien pueden declararse en una sentencia, asignarse valores más adelante.`

Comment (thecodingaviator) [reply guide/spanish/java/variables/index.md]: Thanks!

**Comment by thecodingaviator** [comment]
Thank you for your contribution to the page! 👍
We're happy to accept these changes, and look forward to future contributions. 📝

**Pull request merged**